modules/cluster-logging-deploy-cli.adoc

```diff
@@ -267,7 +267,7 @@ spec:
     elasticsearch:
       nodeCount: 3
       storage:
-        storageClassName: "gp2"
+        storageClassName: "<storage-class-name>"
```

Comment:
Is this variable clear enough without the annotation? And will someone on dedicated have this available to specify?

Reply:
Yes. Was going to ask about Dedicated after I got the wording down.